site/content/contribute/server/developer-setup/arch.md

```diff
@@ -21,41 +21,49 @@
     ```
     If you don't migrate your data, the new, docker-compose-managed containers will start out empty. To remove the old containers -- destroying any existing data -- use `make clean-old-docker`.
     
-3. Install Go:
+2. Install docker-compose
+
+    ```sh
+    sudo curl -L "https://github.com/docker/compose/releases/latest/download/docker-compose-$(uname -s)-$(uname -m)" -o /usr/local/bin/docker-compose
+    sudo chmod +x /usr/local/bin/docker-compose
+    ```
+
+
+4. Install Go:
 
     ```sh
     sudo pacman -S base-devel
     sudo pacman -S go
     ```
 
-4. Update your shell's initialization script (e.g. `.bashrc` or `.zshrc`) and add the following:
+5. Update your shell's initialization script (e.g. `.bashrc` or `.zshrc`) and add the following:
```

Comment:
I don't know if the list numbers go correctly on these, but they look fine on preview.

Reply:
The hugo server usually ignores the numbering of the source and just counts incrementally. So in the source you could number a list like this:

```md
12.   
999.    
1.   
```
and hugo would still output:

```md
1.
2.
3.
```

I only took the existing list, inserted my step and applied +1 on every following step. I think however, that the numbering in all the edited files is slightly off.

Maybe we could address that in a separate issue? Or I could add a commit renumbering all the steps in a linear sequence to this PR.